.github/workflows/update_aas_extension.yml

```diff
@@ -0,0 +1,92 @@
+name: Update extension on AAS, triggered by dd-trace-dotnet nightly build
```

Comment:
NIT - as discussed could be better to have this in the gitlab pipeline. As we would have the dd-trace-dotnet gitlab thee rel env and aas pipelines on nightly builds. But clearly not an issue for now